### DISCUSSION TITLE
Creazione cliente automaticamente dalla fattura

### DISCUSSION THREAD

**Discussion by antonelloventurino** [fattureincloud/api#209] [Q&A] [answered]
Salve,
ho notato che con le nuove API il cliente non viene più creato automaticamente dai dati della fattura (durante l'invio della stessa). Ho cercato nella documentazione e non mi sembra ci sia nessuna variabile da impostare per farlo. Devo fare per forza due chiamate API? O mi sfugge qualcosa?

Grazie :)

**Comment by emajo** [comment, marked as answer]
Ciao @antonelloventurino, sì, attualmente questa funzionalità non è presente sulle API v2 per cui è necessario fare le due chiamate, non sappiamo se e eventualmente quando questa feature verrà implementata.

Grazie,
Emanuele.
